src/PostgREST/OpenAPI.hs

```diff
@@ -185,7 +185,9 @@ makePathItem (t, cs, _) = ("/" ++ unpack tn, p $ tableInsertable t)
   where
     tOp = (mempty :: Operation)
       & tags .~ Set.fromList [tn]
-      & description .~ tableDescription t
+      -- Use first line of table description as summary; rest as description (if present)
+      & summary .~ (strip . takeWhile (/='\n') <$> tableDescription t)
```

Comment:
It does sound like a more suitable operator. Don't forget we also need to trip that `\n` off the start of `tDesc` too.

I tried implementing this but failed: I just don't know enough haskell. My attempt: http://sprunge.us/ZSce